### DISCUSSION TITLE
Issues with joint analyses

### DISCUSSION THREAD

**Discussion by QRemy** [gammapy/gammapy#5307] [General]
**Estimators**
- flux points
[#4988](https://github.com/gammapy/gammapy/issues/4988)Allow estimators to work with different instruments
- flux maps
[#5289](https://github.com/gammapy/gammapy/pull/5289) Add "profile" method for combine_flux_maps 
[#5285](https://github.com/gammapy/gammapy/pull/5285) Add support for stat_scan in TSMapEstimator
- sensitivity
[#4712](https://github.com/gammapy/gammapy/issues/4712)Adapt SensitivityEstimator to support MapDataset and joint datasets
[#4576](https://github.com/gammapy/gammapy/issues/4576)Generic sensitivity estimation

**Cumulative significane:**
`display(M87_datasets.info_table(cumulative=True))`
`ValueError: Missing exposure map for EDispKernelMap.stack`

https://github.com/gammapy/gammapy/blob/ddf72e06a15cf995363c86d1acb9dce4eabdbb91/gammapy/datasets/core.py#L524-L530

-> only a specific problem with to hawc IRFs as they don't have exposure ? can we do something in that case ?

**Comment by bkhelifi** [comment]
Where do we stand @QRemy ?
